File: 0000_50/26/main.dart
class Solution {
  int removeDuplicates(List<int> nums) {
    if (nums.length == 1) {
      return 1;
    }
    int j = 1;
    for (int i = 1; i < nums.length; i++) {
      if (nums[i] != nums[i - 1]) {
        nums[j] = nums[i];
        j++;
      }
    }
    return j;
  }
}

void main() {
  Solution s = Solution();
  List<int> nums = [0, 0, 1, 1, 1, 2, 2, 3, 3, 4];
  print(s.removeDuplicates(nums));
}

File: 0000_50/26/main.py
from typing import List

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l = len(nums)
        if l == 1:
            return 1
        j = 1
        for i in range(1, l):
            if nums[i] != nums[i - 1]:
                nums[j] = nums[i]
                j += 1

        return j